appAuthHelperFetchTokens.js

```diff
@@ -73,7 +73,14 @@
             });
             break;
         case "appAuth-getFreshAccessToken":
-            tokenManager.silentAuthzRequest(e.data.resourceServer);
+            tokenManager.silentAuthzRequest(e.data.resourceServer).then((strategyUsed) => {
+                if (strategyUsed === "refreshToken") {
```

Comment:
Just wondering, if 'authCode' strategy is not considered (as a condition), why can't this be a part of the function defined in TokenManager.js#184? 

As a matter of fact, if this returned value is not used anywhere else, maybe it doesn't need to be returned at all (as well as 'authCode' one)?

Reply:
Yes, I agree it's not the prettiest code path. The reason I had to have it here is because this file is essentially the "broker" that exists to handle message passing between the parent/child frames and the "meat" of the logic in TokenManager.js. Simply due to the nature of the flows involved, the broker will be engaged at pretty different points. I don't think there would be any kind of straightforward refactoring possible to get the two strategies to be more "aligned". So I think I'll live with this little "sin" for now.